src/it/java/org/eclipse/microprofile/starter/utils/MPSpec.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2017-2020 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package org.eclipse.microprofile.starter.utils;
+
+/**
+ * @author Michal Karm Babacek <karm@redhat.com>
+ */
+public enum MPSpec {
+    DEFAULT(new String[][]{
+            new String[]{"data/hello", "Hello World"}
+    }),
+    CONFIG(new String[][]{
+            new String[]{"data/config/injected", "Config value as Injected by CDI Injected value"},
+            new String[]{"data/config/lookup", "Config value from ConfigProvider lookup value"}
+    }),
+    FAULT_TOLERANCE(new String[][]{
+            new String[]{"data/resilience", "Fallback answer due to timeout"}
+    }),
+    HEALTH_CHECKS(new String[][]{
+            new String[]{"health", "\"UP\""}
```

Comment:
also the new verison of Health has /health/live and /health/ready. We also need to check the two additional end points.

Reply:
@rdebusscher @Emily-Jiang 

I am resolving this with this comment in the code:

```
    // Note that there are runtimes ATTOW, such as Tomee, that do not do latest MP spec
    //  and thus cannot do /health/ready endpoint. We stick to the largest common denominator here.
    HEALTH_CHECKS(new String[][]{
            new String[]{"health", "\"UP\""}
    }),
```

I suggest we don't block the PR on this and move forward.  I can add more MP/Runtime specific stuff later if necessary.  Although I hope runtimes will get updated to the latest MP :smiley: